integration-test/src/main/java/com/sequenceiq/it/cloudbreak/cloud/v4/azure/AzureProperties.java

```diff
@@ -300,6 +315,16 @@ public static class Network {
 
         private Set<String> subnetIds;
 
+        private String databasePrivateDnsZoneId;
```

Comment:
Please introduce this new application parameter to: 
```
integrationstes:
...
  azure:
    ...
    network:
      networkId: 
      noPublicIp:
      resourceGroupName: 
      subnetIds:
        - 
        - 
      databasePrivateDnsZoneId:
```
in [application.yml](https://github.com/hortonworks/cloudbreak/blob/master/integration-test/src/main/resources/application.yml#L156-L162).